nodelet/include/nodelet/nodelet.h

```diff
@@ -127,6 +183,10 @@ class Nodelet
   // Protected data fields for use by the subclass.
 protected:
   inline const std::string& getName() const { return nodelet_name_; }
+  inline std::string getSuffixedName(const std::string& suffix) const
+  {
+    return nodelet_name_ + std::string(".") + suffix;
```

Comment:
As mentioned in my previous comment wrapping the string in `std::string` is not necessary.